hq_superset/utils.py

```diff
@@ -41,3 +45,26 @@ def get_role_name_for_domain(domain):
     # Prefix in-case domain name matches with known role names such as admin
     # Same prefix pattern as schema only by coincidence, not a must.
     return f"{DOMAIN_PREFIX}{domain}"
+
+
+def get_column_dtypes(datasource_defn):
+    """
+    Maps UCR column data types to Pandas data types.
+
+    See corehq/apps/userreports/datatypes.py for possible data types.
+    """
+    # TODO: How are array indicators handled in CSV export?
+
+    COL_ATTR = 'column_id'
+    pandas_dtypes = {
+        'date': 'datetime64[ns]',
+        'datetime': 'datetime64[ns]',
+        'string': 'string',
+        'integer': 'int64',
+        'decimal': 'float64',
+        'small_integer': 'int8',  # TODO: Is this true?
```

Comment:
According to this IBM document a small_integer is 2 bytes, although I'm not sure if the Dimagi dev who decided to call it "small_integer" followed this standard. I wonder what data will be stored in the column with this type?

I see in HQ it's used [here](https://github.com/dimagi/commcare-hq/blob/ea1f477742ada5ba7898c21522dac4e71f70e779/corehq/apps/aggregate_ucrs/column_specs.py#L338), so I'm assuming that this datatype will contain a 1 or a 0?